### PR TITLE
feat: add ability to parse addr infos from strings

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Will be used to parse bootstrappers from a comma separated list. 

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: cmd/qgb/helpers/parse.go
Comment (evan-forbes) [inline cmd/qgb/helpers/parse.go:17]: [non blocking question]
dereferencing this value makes sense to me, as I never like using pointers when we don't strictly need to (like for a read only struct such as `AddrInfo`), especially since its already a slice. However, I'm uncertain if there is a reason why we need a pointer, since the maintainers return a pointer. Do we know why they use a pointer?

**Review by evan-forbes — APPROVED** [review]

File: cmd/qgb/helpers/parse.go
Comment (SweeXordious) [reply cmd/qgb/helpers/parse.go:17]: I took the implementation from Celestia-node :cat:

I guess the reason for using pointers is that the addresses could get heavy and we don't want to copy them every time. 

What dl you think? Should i switch to using pointers?

Comment (SweeXordious) [reply cmd/qgb/helpers/parse.go:17]: I took the implementation from Celestia-node :cat:

I guess the reason for using pointers is that the addresses could get heavy and we don't want to copy them every time. 

What dl you think? Should i switch to using pointers?

Comment (evan-forbes) [reply cmd/qgb/helpers/parse.go:17]: nah, if celestia-node uses it and they don't have issues its probably fine. It should be an improvement to not use pointers, as it makes no sense to me, but I'm just paranoid lol.

Comment (SweeXordious) [reply cmd/qgb/helpers/parse.go:17]: Shit, after merging I remembered that I did couple changes, here is the original implementation:
https://github.com/celestiaorg/celestia-node/blob/bd96b3dab06e7d733cf0e62adb94802e2007a407/nodebuilder/p2p/bootstrap.go#L45-L64

But internally, it's almost the same functions called. Do I revert? or we keep it until we have an issue?

**Pull request merged**
